docs/docs/references/preview-projects.mdx

```diff
@@ -0,0 +1,173 @@
+
+You have three options for setting up project connections in your GitHub Actions:
+1. set up a single project connection that's used in every pull request. E.g. always use the `prod` target. You can [follow the instructions here](/guides/cli/how-to-use-lightdash-preview#dbt_profiles) for your data warehouse
+2. if you're using dbt Cloud, you can [use the dbt Cloud schema](/guides/cli/how-to-use-lightdash-preview#how-to-use-the-dbt-cloud-schema-in-your-preview-project) to manage the connection used in the pull request.
+3. [create a project connection per developer in GitHub](/guides/cli/how-to-use-lightdash-preview#how-to-use-the-developer-credentials-in-your-preview-project) (similar to `dev` profiles in your dbt project) so the connection changes depending on who opened the pull request. Tis is useful if you use different development schemas for team members developing in dbt.
+
+## Content in preview projects
+
+When you create a preview project, the content is copied from the production project. The content in a preview project does not get re-synced from the project it's been created from. 
+
+For example:
+- I have a project called `Lightdash Analytics` with a chart called `Revenue by source` 
+- I create a preview of that project called `Preview Lightdash Analytics`
+- The `Preview Lightdash Analytics` project has the same chart called `Revenue by source` 
+- I change the `Revenue by source` chart in `Lightdash Analytics` (i.e. in the production project)
+- The `Revenue by source` chart in `Preview Lightdash Analytics` is not updated. It is the same chart as when the preview project was first created. 
+
+### Promoting content from preview projects
+
+Promoting content allows you to sync charts and dashboards that you've created or updated in your preview project to your production project. This means you can use preview projects to make changes to important content in production without any downtime.
+
+Learn more about promoting content from preview projects [here](/guides/how-to-promote-content).
+
+## Deleting a Preview Project
+
+Preview projects created using any method can be deleted in the [Lightdash App](/references/preview-projects#lightdash-app-2). 
+
+There are also other more automated ways of deleting preview projects created in the Lightdash CLI and in GitHub Actions explained below.
+
+### Lightdash CLI Tool (Local)
+
+If you use the command `lightdash preview` to generate a preview project from your CLI, then hitting `Enter` from your CLI should delete the preview project. 
+
+If you use the command `lightdash start-preview` to generate a preview project from your CLI, then running the command `lightdash stop-preview` should delete your preview project. 
+
+You can also delete preview projects created from the CLI in the [Lightdash App](/references/preview-projects#lightdash-app-2). 
```

Comment:
@TuringLovesDeathMetal i would delete this line - you already said it in the intro of this section and if the reader keeps going another couple of lines it says this again

Reply:
ya, I thought it might be repetitive, but also wanted to make sure they actually see it (I normally just scroll using the sidebar navigation, so I'd miss the first line). 

Will remove though!